.drone.yml

```diff
@@ -20,26 +20,26 @@ steps:
   - name: Sanity and Cleanup
     commands:
       - if [ -d /tmp/${DRONE_BUILD_NUMBER}/babylontest ] ; then rm -rf /tmp/${DRONE_BUILD_NUMBER}/babylontest; fi
-  - name: Build Babylon
+  - name: Build bbi
     commands:
       - mkdir -p /tmp/${DRONE_BUILD_NUMBER} ; git rev-parse HEAD > /tmp/${DRONE_BUILD_NUMBER}/bbi_hash
       - go build -o bbi cmd/bbi/main.go
       - ./bbi version
-  - name: Get BabylonTest
+  - name: Get babylontest
     environment:
-      BABYLONTEST_BRANCH: master
+      babylontest_BRANCH: master
```

Comment:
Even though this is purely for drone, it may make sense to just refer to this ENV as bbitest_branch instead